tfh/usr/share/doc/tfh/tfh_run_apply.md

```diff
@@ -0,0 +1,23 @@
+## `tfh run apply`
+
+Confirm a pending plan to allow the run to proceed.
```

Comment:
Elsewhere you're specific about this doing an apply, so I'd be that specific here too. Also, I believe `pending` is not yet planned, and `planned` is the state that's confirmable / apply-able.

> Confirm a run that has planned to begin an apply.